config/environments/development.rb

```diff
@@ -40,11 +46,17 @@
   config.assets.quiet = true
 
   # Raises error for missing translations.
-  config.action_view.raise_on_missing_translations = true
+  config.i18n.raise_on_missing_translations = true
+
+  # Annotate rendered view with file names.
+  # config.action_view.annotate_rendered_view_with_filenames = true
```

Comment:
We've switched this on in the other Rails 6.1 upgrades.  It may be a useful feature for development?